src/EventHandler/AbstractMessage.php

```diff
@@ -208,4 +209,48 @@ public function reply(
             updateStickersetsOrder: $updateStickersetsOrder
         );
     }
+
+    /**
+     * Reply inlineBotResult to message
+     *
+     * @param BigInteger $queryId Query ID from <a href='https://docs.madelineproto.xyz/API_docs/methods/messages.getInlineBotResults.html'>messages.getInlineBotResults</a>
+     * @param int $id Result ID from <a href='https://docs.madelineproto.xyz/API_docs/methods/messages.getInlineBotResults.html'>messages.getInlineBotResults</a>
+     * @param array|null $replyTo reply to specific message
+     * @param int|string|null $sendAs Send this message as the specified peer
+     * @param int|null $scheduleDate Schedule date.
+     * @param bool $hideVia Whether to hide the via @botname in the resulting message (only for bot usernames encountered in the <a href='https://docs.madelineproto.xyz/API_docs/constructors/config.html'>config</a>)
+     * @param bool $clearDraft Clears the draft field
+     * @param bool $silent Whether to send the message silently, without triggering notifications.
+     * @param bool $background Send this message as background message
+     * @return Update|null Return Update or null
+     *
+     */
+    public function replyInlineBotResult(
+        BigInteger $queryId,
+        int $id,
+        ?array $replyTo = null,
+        int|string|null $sendAs = null,
+        ?int $scheduleDate = null,
+        bool $hideVia = false,
+        bool $clearDraft = false,
+        bool $silent = false,
+        bool $background = false
+    ): ?Update
+    {
+        return $this->API->wrapUpdate($this->API->methodCallAsyncRead(
```

Comment:
This requires a manual call to getInlineBotResults before, this is not good: it's better to provide something like a `getInlineBotResults` method which returns an array of `InlineResult` objects which can then be sent using `->send()`.

Reply:
ahhhh yes its better!